    public ExceptionEvent(Object source, Throwable exception) {
        super(source, getMessageFromException(exception), -1);
        this.exception = exception;
    }

